externs/browser/w3c_geolocation.js

```diff
@@ -27,8 +27,18 @@
 function Geolocation() {}
 
 /**
- * @param {function(!GeolocationPosition)} successCallback
- * @param {(function(!GeolocationPositionError)|null)=} opt_errorCallback
+ * @typedef {function(!Position): void}
```

Comment:
I see in you original PR #3297 that you renamed GeolocationPosition to Position. Do you still have a PR for the renaming, since this new name won't be recognized.

Reply:
Not at this stage, this was just stupidity on my part. I hadn't noticed the TravisCI build failure before I generated the PR. Anyhoo I will force push the fix